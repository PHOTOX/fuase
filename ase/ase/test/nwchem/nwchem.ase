dict(basis='3-21G',
     basispar=None,
     bq=None,
     charge=-1,
     charges=None,
     convergence={'gradient': None, 'energy': None, 'lshift': None, 'density': None},
     ecp=None,
     geometry='nocenter noautosym noautoz',
     initial_magmoms=[0.0, 0.0, 0.0, 0.0, 0.0],
     odft=False,
     raw='',
     smearing=None,
     so=None,
     spinorbit=False,
     task='gradient',
     xc='LDA')
