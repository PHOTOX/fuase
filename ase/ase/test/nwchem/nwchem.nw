charge -1
geometry nocenter noautosym noautoz
  H -1.02018181669 0.0 -0.0776138136391
  O 0.0 0.0 0.0
  H -0.00414365744789 0.0 1.37212327489
  O 0.0 0.0 2.74
  H 1.02168476516 0.0 2.80359419428
end
start
basis
  * library 3-21G
end

dft
  xc slater pw91lda
  mult 1
end

task dft gradient
