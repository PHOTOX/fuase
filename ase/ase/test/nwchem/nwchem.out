 argument  1 = nwchem.nw
                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.3
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2012
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = aeris
    program         = nwchem
    date            = Fri Oct 17 08:37:53 2014

    compiled        = Thu_Sep_25_10:11:34_2014
    source          = /build/nwchem-ispQuo/nwchem-6.3+r2
    nwchem branch   = 6.3
    nwchem revision = 24652
    ga revision     = 10379
    input           = nwchem.nw
    prefix          = nwchem.
    data base       = ./nwchem.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 H                    1.0000    -1.02018182     0.00000000    -0.07761381
    2 O                    8.0000     0.00000000     0.00000000     0.00000000
    3 H                    1.0000    -0.00414366     0.00000000     1.37212327
    4 O                    8.0000     0.00000000     0.00000000     2.74000000
    5 H                    1.0000     1.02168477     0.00000000     2.80359419

      Atomic Mass 
      ----------- 

      H                  1.007825
      O                 15.994910


 Effective nuclear repulsion energy (a.u.)      30.3944859818

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
    -0.0049902164     0.0000000000    49.1670866514


            XYZ format geometry
            -------------------
     5
 geometry
 H                    -1.02018182     0.00000000    -0.07761381
 O                     0.00000000     0.00000000     0.00000000
 H                    -0.00414366     0.00000000     1.37212327
 O                     0.00000000     0.00000000     2.74000000
 H                     1.02168477     0.00000000     2.80359419

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 O                |   1 H                |     1.93344  |     1.02313
    5 H                |   4 O                |     1.93444  |     1.02366
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



  warning:::::::::::::: from_nwchemrc
  NWCHEM_BASIS_LIBRARY set to: </opt/build/nwchem-src-2013-02-06/QA/../src/basis/libraries/>
  but file does not exist !
  using compiled in library


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           3-21G                    on all atoms 



                                 NWChem DFT Module
                                 -----------------


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.44717800E+00  0.156285
  1 S  8.24547000E-01  0.904691

  2 S  1.83192000E-01  1.000000

  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.22037000E+02  0.059239
  1 S  4.84308000E+01  0.351500
  1 S  1.04206000E+01  0.707658

  2 S  7.40294000E+00 -0.404453
  2 S  1.57620000E+00  1.221560

  3 P  7.40294000E+00  0.244586
  3 P  1.57620000E+00  0.853955

  4 S  3.73684000E-01  1.000000

  5 P  3.73684000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           3-21G                   2        2   2s
 O                           3-21G                   5        9   3s2p


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     5
          No. of electrons :    20
           Alpha electrons :    10
            Beta electrons :    10
          Charge           :    -1
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:    24
                     number of shells:    16
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
            Perdew 1991 LDA Correlation Functional  1.000 local    

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          H                   0.35       45           9.0       434
          O                   0.60       49           8.0       434
          Grid pruning is: on 
          Number of quadrature shells:   233
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -150.22597622

 Renormalizing density from      19.00 to     20

      Non-variational initial energy
      ------------------------------

 Total energy =    -154.633012
 1-e energy   =    -281.379815
 2-e energy   =      96.352316
 HOMO         =       0.048850
 LUMO         =       0.468608

   Time after variat. SCF:      0.0
   Time prior to 1st pass:      0.0

 #quartets = 7.729D+03 #integrals = 1.937D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./nwchem.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      3        Max. records in file   = 506068
 No. of bits per label  =      8        No. of bits per value  =     64


 Grid_pts file          = ./nwchem.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     38        Max. recs in file   =   2698814


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.44            12442164
          Stack Space remaining (MW):       13.11            13106926

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1   -150.1511980603 -1.81D+02  2.71D-02  6.83D-01     0.3
 d= 0,ls=0.0,diis     2   -150.1955518416 -4.44D-02  1.11D-02  1.95D-01     0.4
 d= 0,ls=0.0,diis     3   -150.2029063270 -7.35D-03  8.17D-03  7.56D-02     0.5
 d= 0,ls=0.0,diis     4   -150.2096631556 -6.76D-03  1.84D-03  2.93D-03     0.6
 d= 0,ls=0.0,diis     5   -150.2100106127 -3.47D-04  2.58D-04  5.88D-05     0.7
  Resetting Diis
 d= 0,ls=0.0,diis     6   -150.2100157720 -5.16D-06  9.15D-05  1.16D-05     0.8
 d= 0,ls=0.0,diis     7   -150.2100170011 -1.23D-06  9.24D-06  1.25D-08     1.0
 d= 0,ls=0.0,diis     8   -150.2100169998  1.33D-09  4.62D-06  3.34D-08     1.1


         Total DFT energy =     -150.210016999774
      One electron energy =     -275.374022329533
           Coulomb energy =      112.174889407190
    Exchange-Corr. energy =      -17.405370059198
 Nuclear repulsion energy =       30.394485981767

 Numeric. integr. density =       20.000000536168

     Total iterative time =      1.0s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-1.812430D+01
              MO Center=  1.6D-04, -9.0D-13,  2.4D+00, r^2= 8.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    14      0.912474  4 O  s                  3      0.360640  2 O  s          

 Vector    2  Occ=2.000000D+00  E=-1.812310D+01
              MO Center= -1.6D-04, -1.1D-13,  3.7D-01, r^2= 8.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.912355  2 O  s                 14     -0.360341  4 O  s          

 Vector    3  Occ=2.000000D+00  E=-5.189761D-01
              MO Center=  9.3D-03,  2.7D-10,  1.4D+00, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    19      0.482961  4 O  s                  8      0.463041  2 O  s          
    14     -0.161077  4 O  s                  3     -0.154603  2 O  s          

 Vector    4  Occ=2.000000D+00  E=-4.971257D-01
              MO Center= -1.0D-02,  3.4D-10,  1.3D+00, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.511184  2 O  s                 19     -0.492671  4 O  s          
     3     -0.166324  2 O  s                 14      0.160068  4 O  s          
     4      0.150826  2 O  s          

 Vector    5  Occ=2.000000D+00  E=-7.535605D-02
              MO Center= -1.4D-04,  4.6D-11,  1.4D+00, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    13      0.352582  3 H  s                 24     -0.205219  5 H  s          
     2     -0.203769  1 H  s                 16     -0.199914  4 O  px         
     5      0.196602  2 O  px                 7      0.180672  2 O  pz         
    18     -0.180098  4 O  pz                20     -0.175124  4 O  px         
     9      0.171913  2 O  px                11      0.161055  2 O  pz         

 Vector    6  Occ=2.000000D+00  E=-3.697390D-02
              MO Center=  6.9D-04,  1.0D-10,  1.4D+00, r^2= 2.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    19     -0.369987  4 O  s                  8      0.365459  2 O  s          
    16      0.262460  4 O  px                 5      0.260174  2 O  px         
    20      0.255006  4 O  px                 9      0.252648  2 O  px         
    24      0.250948  5 H  s                  2     -0.249071  1 H  s          
     1     -0.167276  1 H  s                 23      0.167797  5 H  s          

 Vector    7  Occ=2.000000D+00  E= 1.918245D-02
              MO Center= -1.3D-03,  1.6D-10,  1.4D+00, r^2= 2.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.369019  2 O  s                 19      0.364608  4 O  s          
    22      0.237091  4 O  pz                11     -0.229845  2 O  pz         
     9      0.199471  2 O  px                18      0.200041  4 O  pz         
    20     -0.194391  4 O  px                 5      0.193287  2 O  px         
     7     -0.193304  2 O  pz                13     -0.191491  3 H  s          

 Vector    8  Occ=2.000000D+00  E= 1.182531D-01
              MO Center=  5.5D-04,  4.2D-11,  1.4D+00, r^2= 2.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      0.446379  2 O  pz                22      0.443924  4 O  pz         
     7      0.366876  2 O  pz                18      0.364509  4 O  pz         

 Vector    9  Occ=2.000000D+00  E= 1.284197D-01
              MO Center= -9.7D-19, -5.2D-10,  1.5D+00, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    21      0.474739  4 O  py                10      0.431962  2 O  py         
    17      0.376994  4 O  py                 6      0.342191  2 O  py         

 Vector   10  Occ=2.000000D+00  E= 1.359767D-01
              MO Center= -4.0D-19, -5.9D-10,  1.2D+00, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      0.471900  2 O  py                21     -0.428716  4 O  py         
     6      0.382334  2 O  py                17     -0.348198  4 O  py         

 Vector   11  Occ=0.000000D+00  E= 3.225910D-01
              MO Center=  7.0D-03, -4.7D-11,  1.4D+00, r^2= 4.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    24      0.810232  5 H  s                  2      0.804355  1 H  s          
    19     -0.547305  4 O  s                  8     -0.544431  2 O  s          
    20     -0.395290  4 O  px                 9      0.392736  2 O  px         
    16     -0.212656  4 O  px                 5      0.211057  2 O  px         
    13      0.205727  3 H  s          

 Vector   12  Occ=0.000000D+00  E= 3.532092D-01
              MO Center= -5.2D-03, -2.6D-11,  1.4D+00, r^2= 4.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.876974  1 H  s                 24     -0.871966  5 H  s          
     8     -0.483288  2 O  s                 19      0.477990  4 O  s          
     9      0.437100  2 O  px                20      0.435447  4 O  px         
     5      0.249934  2 O  px                16      0.249018  4 O  px         

 Vector   13  Occ=0.000000D+00  E= 4.787011D-01
              MO Center= -4.4D-03, -2.8D-11,  1.4D+00, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    13      1.098242  3 H  s                 11     -0.451617  2 O  pz         
    22      0.450107  4 O  pz                 2     -0.410199  1 H  s          
    24     -0.408335  5 H  s                  7     -0.302986  2 O  pz         
    18      0.302033  4 O  pz                12      0.268128  3 H  s          
    20      0.215217  4 O  px                 9     -0.212801  2 O  px         

 Vector   14  Occ=0.000000D+00  E= 1.152925D+00
              MO Center= -1.2D-02,  6.9D-11,  1.3D+00, r^2= 3.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.934107  1 H  s                 23     -0.924766  5 H  s          
    24      0.679420  5 H  s                  2     -0.673853  1 H  s          
     5      0.187459  2 O  px                16      0.180895  4 O  px         

 Vector   15  Occ=0.000000D+00  E= 1.178220D+00
              MO Center=  2.8D-02, -5.3D-11,  1.4D+00, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    23      0.734468  5 H  s                 12      0.714384  3 H  s          
     1      0.700936  1 H  s                 13     -0.651616  3 H  s          
    24     -0.463420  5 H  s                  2     -0.438203  1 H  s          
    16     -0.241606  4 O  px                 5      0.235616  2 O  px         
    19      0.197697  4 O  s                  8      0.193726  2 O  s          

 Vector   16  Occ=0.000000D+00  E= 1.230833D+00
              MO Center= -2.1D-02, -8.1D-11,  1.3D+00, r^2= 2.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    13      1.566216  3 H  s                 12     -1.059497  3 H  s          
     2     -0.696008  1 H  s                 24     -0.677903  5 H  s          
     1      0.593409  1 H  s                 23      0.565765  5 H  s          
    11     -0.217979  2 O  pz                19     -0.218733  4 O  s          
    22      0.218260  4 O  pz                 8     -0.215574  2 O  s          

 Vector   17  Occ=0.000000D+00  E= 1.653971D+00
              MO Center= -2.0D-18, -1.1D-09,  1.4D+00, r^2= 2.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      0.739158  4 O  py                 6      0.723140  2 O  py         
    21     -0.681181  4 O  py                10     -0.666098  2 O  py         

 Vector   18  Occ=0.000000D+00  E= 1.684473D+00
              MO Center= -8.5D-18, -2.2D-09,  1.3D+00, r^2= 2.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.738218  2 O  py                17     -0.722115  4 O  py         
    10     -0.691578  2 O  py                21      0.676807  4 O  py         

 Vector   19  Occ=0.000000D+00  E= 1.738712D+00
              MO Center=  2.0D-03,  7.8D-11,  1.4D+00, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.653420  2 O  pz                18      0.653129  4 O  pz         
    11     -0.631369  2 O  pz                22     -0.632853  4 O  pz         
     9      0.388030  2 O  px                20      0.365322  4 O  px         
     5     -0.325936  2 O  px                16     -0.305254  4 O  px         
     1      0.163405  1 H  s                 23     -0.159161  5 H  s          

 Vector   20  Occ=0.000000D+00  E= 1.781224D+00
              MO Center= -6.0D-03,  5.9D-10,  1.4D+00, r^2= 2.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    20      0.798752  4 O  px                 9      0.769906  2 O  px         
    16     -0.648464  4 O  px                 5     -0.624619  2 O  px         
    11      0.355244  2 O  pz                 7     -0.340203  2 O  pz         
    22      0.340544  4 O  pz                18     -0.325772  4 O  pz         
    24     -0.227115  5 H  s                  2      0.219042  1 H  s          


 center of mass
 --------------
 x =  -0.00014364 y =   0.00000000 z =   2.58827751

 moments of inertia (a.u.)
 ------------------
         229.351432242452           0.000000000000         -10.586485076169
           0.000000000000         236.854023666327           0.000000000000
         -10.586485076169           0.000000000000           7.502591423876

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0     -1.000000    -10.000000    -10.000000     19.000000

     1   1 0 0     -0.000598      0.002196      0.002196     -0.004990
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1     -2.589797    -25.878442    -25.878442     49.167087

     2   2 0 0     -8.624963     -8.034652     -8.034652      7.444340
     2   1 1 0     -0.000000     -0.000000     -0.000000      0.000000
     2   1 0 1      2.366163     -4.062605     -4.062605     10.491373
     2   0 2 0    -10.838456     -5.419228     -5.419228      0.000000
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2    -25.581608   -137.438244   -137.438244    249.294880


 Parallel integral file used       1 records with       0 large values



                            NWChem DFT Gradient Module
                            --------------------------



  charge          =  -1.00
  wavefunction    = closed shell



                         DFT ENERGY GRADIENTS

    atom               coordinates                        gradient
                 x          y          z           x          y          z
   1 H      -1.927864   0.000000  -0.146669    0.000033  -0.000000  -0.000339
   2 O       0.000000   0.000000   0.000000   -0.000140  -0.000000  -0.029380
   3 H      -0.007830   0.000000   2.592937   -0.000024   0.000000   0.000185
   4 O       0.000000   0.000000   5.177849   -0.000238  -0.000000   0.029520
   5 H       1.930704   0.000000   5.298025    0.000369   0.000000   0.000015

                 ----------------------------------------
                 |  Time  |  1-e(secs)   |  2-e(secs)   |
                 ----------------------------------------
                 |  CPU   |       0.00   |       0.03   |
                 ----------------------------------------
                 |  WALL  |       0.00   |       0.03   |
                 ----------------------------------------

 Task  times  cpu:        1.4s     wall:        1.4s
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  165      165     1.25e+04  901     6458        0        0      798     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             3.62e+06 6.50e+05 1.58e+06 0.00e+00 0.00e+00 6.38e+03
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 179712 bytes
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        23	        45
	current total bytes		         0	         0
	maximum total bytes		   5320296	  22510840
	maximum total K-bytes		      5321	     22511
	maximum total M-bytes		         6	        23


                                NWChem Input Module
                                -------------------





                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
     F. Aquino, S. Hirata, M. T. Hackler, J. Mullin, P. Nichols, R. Peverati,
    J. Pittner, Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. Silverstein,
    D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
     H. Fruchtl, J. Garza, K. Hirao, R. Kendall, J. A. Nichols, K. Tsemekhman,
      K. Wolinski, J. Anchell, D. Bernholdt, P. Borowski, T. Clark, D. Clerc,
      H. Dachsel, M. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
      A. Hess, J. Jaffe, B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                                A. Wong, Z. Zhang.

 Total times  cpu:        1.4s     wall:        1.4s
